### ISSUE TITLE
Checkboxes in package manager list are jumping

### ISSUE THREAD

**Issue by mnonnenmacher** [eclipse-apoapsis/ort-server#1377]
When clicking the checkboxes in the form to select the package managers for a run, their position is changing:

![image](https://github.com/user-attachments/assets/8360e506-ed22-467e-89af-17e65cc386a6)

![image](https://github.com/user-attachments/assets/a72817b1-ba09-444b-b5c0-410236ddb378)

**Comment by mnonnenmacher** [comment]
@Etsija Thanks for looking into this, I should have mentioned the browser I'm using. What confuses me is that it works when the checkbox is clicked, so maybe some different layout is used if the caret on the right is rendered which avoids the bug?

**Comment by Etsija** [comment]
Yes, there is a different component (`Accordion`) rendered on the right, when the checkbox is clicked, and the caret is part of that. However, as I said, I couldn't find anything wrong with the DOM here, and when it's working perfectly on Chrome, I am running out of options how to fix, if we want to keep this layout.

**Comment by sschuberth** [comment]
FYI, @jexsie, this might be something you want to have a look at, for a start.
